NoComponentDrag.py

```diff
@@ -143,6 +137,10 @@ def is_parametric_mode():
         pass
     return False
 
+def cleanUiItem(item):
+    if item:
+        item.deleteMe()
+
```

Comment:
I noticed the same "Set var = ui obj, check if var exists, if so: delete" lines of code in three separate locations so i made it a function that saves the hassle. It should function exactly the same since python vars are passed by ref not val.

Reply:
Seeing as how all objects are from the ui obj i feel like specifying that clarifies the extent of the function since only those objects will have the `deleteMe()` function. I dont know if "delete" is a good descriptor to use since the function only will delete the item should it exist, if it were deleting an item every time it would not have the "`if item:`" so I am thinking `clear_ui_item()` May be preferential IMO.